### ISSUE TITLE
v3.4.28 html widget and text widget dont work on the all sidebars

### ISSUE THREAD

**Issue by ghost** [AlxMedia/hueman#827]
> Hi. I am using Hueman free theme. After last update to v3.4.28 html widget and text widget didnt work on the all sidebars. If I insert shortcode (picture slider) to html widget on sidebar , he didnt work.
![Untitled-1](https://user-images.githubusercontent.com/3038156/64650369-4d737e00-d41f-11e9-830d-c06560d678e6.jpg)

@eri-trabiccolo I have asked the user to share the system infos

**Comment by eri-trabiccolo** [comment]
I'm going to look tomorrow morning.
It worked with my tests, but I might have missed an edge case?!
Anyway we can make my change a little bit less 'intrusive' and remove the check on the WP version. That would account all missed cases.

**Comment by eri-trabiccolo** [comment]
I've tested this. After https://github.com/presscustomizr/hueman/commit/ad3ee624c1e77ef575e6bec5426da90834835a80
1) shortcodes are not processed in html widgets -> this is the standard WP behavior
2) shortocodes are processed in text widgets.

So I'm partially able to reproduce the user's issue: shortocodes do not work in html widgets, they do in text widgets.

I think that, as per my previous comment, we can allow shortcodes in html widgets, as users are used to it with Hueman.
To do this, without reverting https://github.com/presscustomizr/hueman/commit/ad3ee624c1e77ef575e6bec5426da90834835a80 we can just remove the check on the WP version, which means that we'll always add the `do_shortcode` filter callback to the `text_widget` filter hook (which is shared by text and custom html widgets) if not already added by 3rd party code (happened before `after_setup_theme`).
It'll be up to WP (>4.8.0) to remove it for text widgets.
Just to be more clear, all in all we had to do https://github.com/presscustomizr/hueman/commit/ad3ee624c1e77ef575e6bec5426da90834835a80 because of what I think is a WP core bug which removes only the first `do_shortcode` filter callback added to `text_widget`, while, given the nature of the hooks, all of them should be removed.
